core/arch/arm/plat-ls/main.c

```diff
@@ -206,43 +206,3 @@ void main_secondary_init_gic(void)
 {
 	gic_cpu_init(&gic_data);
 }
-
-#ifdef CFG_HW_UNQ_KEY_REQUEST
-
-#include <types_ext.h>
-int get_hw_unique_key(uint64_t smc_func_id, uint64_t in_key, uint64_t size);
```

Comment:
Should `core/arch/arm/plat-ls/ls_hw_unq_key_a64.S` be removed too?

Reply:
Yes, It should be removed. 
Updated the commit.
Thanks @jenswi-linaro